transcendence-app/src/user/user.controller.ts

```diff
@@ -22,6 +23,7 @@ import {
   ApiForbiddenResponse,
   ApiNotFoundResponse,
   ApiOkResponse,
+  ApiProduces,
```

Comment:
It's a bit outside of the scope of this PR, but I have just noticed we don't use the `AvatarFileInterceptor` . Are we planning to use it in the future?

Reply:
In fact we are not using the local File interceptor (which seems strange to me...)
What am I missing?

Reply:
You are right. We don't use file interceptors in this PR. We use the `AvatarFileInterceptor` in #126 to save the avatar on disk and check mime type and size.

Reply:
It's PR !126 for ticket #123. That's confusing.